sources/academy/tutorials/node_js/scraping_from_sitemaps.md

```diff
@@ -9,6 +9,16 @@ import Example from '!!raw-loader!roa-loader!./scraping_from_sitemaps.js';
 
 # How to scrape from sitemaps {#scraping-with-sitemaps}
 
+>Crawlee recently introduced a new feature that allows you to scrape sitemaps with ease. If you are using Crawlee, you can skip the following steps and just gather all the URLs from the sitemap in a few lines of code:
```

Comment:
This feels like unnecesary dating ? When is "recently" ? Also I think this could work better as admonitions instead of blockquote.